tasks/connectors/ms_defender_tvm/ms_defender_tvm.rb

```diff
@@ -184,10 +164,8 @@ def run(opts)
                 submit_count += 1
                 print_debug "#{submit_count} about to upload file"
                 filename = "microsoft_tvm_kdi_#{submit_count}.json"
-                connector_response_json = connector_upload("#{$basedir}/#{output_directory}", filename, kenna_connector_id, kenna_api_host, kenna_api_key, max_retries)
-                print_good "Success!" if !connector_response_json.nil? && connector_response_json.fetch("success")
+                kdi_upload("#{$basedir}/#{output_directory}", filename, @kenna_connector_id, @kenna_api_host, @kenna_api_key, false, 3)
```

Comment:
Is the `print_good()` above performed somewhere else.  If not, should `print_good()` go after `kdi_upload()`?

Reply:
it's inside the newer kdi_upload method.